lib/opendax/vault.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Opendax
+  class Vault
+    def vault_secrets_path
+      'config/vault-secrets.yml'
+    end
+
+    def exec(command)
+      `docker-compose exec vault sh -c '#{command}'`
+    end
+
+    def unseal(keys)
+      keys.each { |key| exec("vault operator unseal #{key}") }
+    end
+
+    def login(root_token)
+      exec("vault login #{root_token}")
+    end
```

Comment:
I'm not sure if we need one-liner method definitions for lines of code we run only once